functions-methods.js

```diff
@@ -28,3 +86,56 @@
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
 // checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function assignment3() {
+  const emails = [
+    "n.eeken@novi.nl",
+    "tessmellink@novi.nl",
+    "n.eekenanovi.nl",
+    "n.eeken@novinl.",
+    "tessmellink@novi,nl",
+  ];
+
+  for (i = 0; i < emails.length; i++) {
+    let isValid = checkEmailValidity(emails[i]);
+
+    console.log(isValid);
+  }
+}
+
+function checkEmailValidity(emailAdress) {
+  let result = true;
+
+  let hasAt = checkCharacter(emailAdress, "@");
+  let hasComma = checkCharacter(emailAdress, ",");
+
+  let hasDotAtTheEnd = checkHasDotAtTheEnd(emailAdress);
+
+  if (!hasAt || hasComma || hasDotAtTheEnd) {
+    result = false;
+  }
+
+  return result;
+}
+
+function checkCharacter(emailAdress, character) {
+  let result = emailAdress.search(character);
+
+  if (result === -1) {
+    return false;
+  }
+
+  return true;
```

Comment:
Deze zou je nog verder kunnen versimpelen
```suggestion
  return emailAdress.search(character) >= 0
```